include/swift/AST/NameLookupTypeIDZone.def

```diff
@@ -68,6 +68,9 @@ SWIFT_REQUEST(NameLookup, SelfBoundsFromWhereClauseRequest,
               SelfBounds(llvm::PointerUnion<const TypeDecl *,
                                             const ExtensionDecl *>),
               Uncached, NoLocationInfo)
+SWIFT_REQUEST(NameLookup, SelfBoundsFromGenericSignatureRequest,
+              SelfBounds(const ExtensionDecl * extDecl),
+              Uncached, NoLocationInfo)
```

Comment:
Can you try making this request and SelfBoundsFromWhereClauseRequest Cached?

Reply:
It is my pleasure! I will do it!